### PR TITLE
8 submitted claims overview backend

### PR DESCRIPTION
**Pull request by likt0r**

### CODE REVIEW

File: .vscode/settings.json
Comment (stephanmax) [inline .vscode/settings.json:1]: Do we need this kind of editor-specific config in the repo?

File: docker-compose.local.yml [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-      MONGODB_AGENDA_PASSWORD: ${MONGODB_AGENDA_PASSWORD}
-      MONGODB_AGENDA_DB: ${MONGODB_AGENDA_DB}
-      MINIO_HOST: ${MINIO_HOST}
-      MINIO_API_PORT: ${MINIO_API_PORT}
-      MINIO_ACCESS_KEY: ${MINIO_ACCESS_KEY}
-      MINIO_SECRET_KEY: ${MINIO_SECRET_KEY}
-      MINIO_REGION: ${MINIO_REGION}
-      MINIO_BUCKET_NAME: ${MINIO_BUCKET_NAME}
-      DATABASE_URL: ${DATABASE_URL}
+  # faktenforum-api:
```
Comment (stephanmax) [inline docker-compose.local.yml:3]: Why does this need to be commented out?

Comment (likt0r) [reply docker-compose.local.yml:3]: So you can comment it in if you want to test the docker image locally.
If not, you start the backend via npm start

File: .vscode/settings.json
Comment (likt0r) [reply .vscode/settings.json:1]: Yep, we don't. I removed it and added it to git ignore

**Pull request merged**